src/js/device/DeviceCommands.js

```diff
@@ -218,12 +218,13 @@ export default class DeviceCommands {
         return state;
     }
 
-    async getAddress(address_n: Array<number>, coinInfo: BitcoinNetworkInfo, showOnTrezor: boolean): Promise<trezor.Address> {
-        const scriptType: trezor.InputScriptType = getScriptType(address_n);
+    async getAddress(address_n: Array<number>, coinInfo: BitcoinNetworkInfo, showOnTrezor: boolean, multisig?: trezor.MultisigRedeemScriptType, scriptType?: trezor.InputScriptType): Promise<trezor.Address> {
+        if (!scriptType) { scriptType = getScriptType(address_n); }
         const response: Object = await this.typedCall('GetAddress', 'Address', {
             address_n,
             coin_name: coinInfo.name,
             show_display: !!showOnTrezor,
+            multisig,
             script_type: scriptType && scriptType !== 'SPENDMULTISIG' ? scriptType : 'SPENDADDRESS', // script_type 'SPENDMULTISIG' throws Failure_FirmwareError
```

Comment:
it will not work because no matter what you set in `scriptType` this condition will exclude 'SPENDMULTISIG'
And the following comment is here for a reason :)

Reply:
I was worried about this :)

I'm not 100% certain what `scriptType` even does downstream from here. Is it even necessary to achieve the goal of this PR (show multisig addresses on device from TrezorConnect)?  Or would it be sufficient to pass `multisig` downstream?

Thanks for your input!

Reply:
Generally speaking `this.typedCall` will pass message to 'trezor-link' where it's going to be transformed to protobuf binary message and sent via transport (bridge > usb cable) to the device.

since both fields (multisig and script_type) are declared in ["GetAddress"](https://github.com/trezor/connect/blob/develop/src/data/messages/messages.json#L539-L580) this message should be accepted